clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/RateSensor.java

```diff
@@ -32,15 +38,27 @@ public double getRate() {
    * Record the rate value.
    */
   public void record() {
-    sensor.record();
+    rateAccumulator++;
+    if (LatencyUtils.getElapsedTimeInMs(lastUpdateTimeInMS) < ACCUMULATION_PERIOD_MS) {
+      return;
+    }
+    sensor.record(rateAccumulator);
+    lastUpdateTimeInMS = System.currentTimeMillis();
```

Comment:
At the rate we call these metric functions, fetching time may be a big overhead too. Unclear that this is a win overall... again, let's hold off on this change.

I think it may be preferable to delete metrics we don't need anymore.